packages/ts/react-crud/test/autocrud.spec.tsx

```diff
@@ -14,15 +17,31 @@ use(chaiDom);
 describe('@hilla/react-crud', () => {
   describe('Auto crud', () => {
     let user: ReturnType<(typeof userEvent)['setup']>;
+    let matchMediaStub: sinon.SinonStub;
 
-    function TestAutoCrud(props: Partial<AutoCrudProps<Person>> = {}) {
-      return <ExperimentalAutoCrud service={personService()} model={PersonModel} {...props} />;
-    }
+    before(() => {
+      matchMediaStub = sinon.stub(window, 'matchMedia');
+      matchMediaStub.returns({
+        matches: false,
+        addListener: () => {},
+        removeListener: () => {},
+        addEventListener: () => {},
+        removeEventListener: () => {},
+      });
+    });
```

Comment:
Since we run tests headless by default we need to stub `matchMedia` for all tests, otherwise `matches` always returns `true`.